store/sqlstore/upgrade.go

```diff
@@ -699,6 +699,9 @@ func UpgradeDatabaseToVersion512(sqlStore SqlStore) {
 
 func UpgradeDatabaseToVersion513(sqlStore SqlStore) {
 	if shouldPerformUpgrade(sqlStore, VERSION_5_12_0, VERSION_5_13_0) {
+		// The previous jobs ran once per minute, cluttering the Jobs table with somewhat useless entries. Clean that up.
```

Comment:
As discussed on community, this won't actually run /for/ community, since we've already upgraded to 5.13. However, once cherry-picked and released, users upgrading will enjoy this cleanup.